Gemfile

```diff
@@ -127,6 +129,7 @@ group :development, :test do
   gem 'brakeman', require: false
   gem 'rubocop'
   gem 'scss-lint'
+  gem 'awesome_print'
```

Comment:
I have added this in the development environment, forgot to remove it. Will remove this in next commits.